Formula/omniorb.rb

```diff
@@ -30,6 +30,7 @@ class Omniorb < Formula
   end
 
   def install
+    ENV["PYTHON"] = which("python3")
```

Comment:
```suggestion
    ENV["PYTHON"] = Formula["python@3.9"].opt_bin/"python3"
```

Reply:
Using `which` should also only pick up the Python dependency.